cmd/utils.go

```diff
@@ -347,6 +351,16 @@ func dockerInspect(ContainerName string, pattern string) string {
 		return parts[1]
 	}
 
+	if pattern == "BindsData" {
```

Comment:
Can you comment what this part is about ?

Reply:
sure